source/beamline/BioXAS/BioXASSSRLMonochromatorEnergyControl.cpp

```diff
@@ -0,0 +1,23 @@
+#include "BioXASSSRLMonochromatorEnergyControl.h"
+
+BioXASSSRLMonochromatorEnergyControl::BioXASSSRLMonochromatorEnergyControl(const QString &name, const QString &readPVname, const QString &writePVname, const QString &movingPVname, const QString &stopPVname, QObject *parent, double tolerance, double moveStartTimeoutSeconds, AMAbstractControlStatusChecker *statusChecker, int stopValue, const QString &description) :
+    AMPVwStatusControl(name, readPVname, writePVname, movingPVname, stopPVname, parent, tolerance, moveStartTimeoutSeconds, statusChecker, stopValue, description)
+{
+
+}
+
+BioXASSSRLMonochromatorEnergyControl::~BioXASSSRLMonochromatorEnergyControl()
+{
+
+}
+
+AMControl::FailureExplanation BioXASSSRLMonochromatorEnergyControl::move(double setpoint)
```

Comment:
Hmmm, as far as I know, this class was existing before this issue was created, and I'm not sure I remember why. I did rename it and am thinking I might move the energy calibration code into it eventually. Is there a problem with it as written?